geode-core/build.gradle

```diff
@@ -271,7 +271,6 @@ dependencies {
   compile('com.healthmarketscience.rmiio:rmiio')
 
   compile(project(':geode-common'))
-  compile(project(':geode-json'))
```

Comment:
Removing this line; will this impact building geode-web-api and geode-pulse?

Reply:
The geode-web-api includes geode-json as a dependency.  There are tickets for getting rid of it there and in geode-pulse.